package/batocera/core/batocera-configgen/configgen/configgen/generators/scummvm/scummvmGenerator.py

```diff
@@ -41,16 +41,20 @@ def generate(self, system, rom, playersControllers, metadata, guns, wheels, game
         with ensure_parents_and_open(scummConfigFile, 'w') as configfile:
             scummConfig.write(configfile)
 
-        # Find rom path
+        # Find rom path and rom name
         if rom.is_dir():
           # rom is a directory: must contains a <game name>.scummvm file
           romPath = rom
           romName = next(rom.glob("*.scummvm")).stem
-        else:
+
+        if rom.is_file():
           # rom is a file: split in directory and file name
           romPath = rom.parent
-          # Get rom name without extension
-          romName = rom.stem
+          # obtain game ID within file
```

Comment:
Does this need to also be applied to the `.scummvm` files if the rom is a directory?

Reply:
I'm not sure what the usecase here is...
if you would have a dir `monkey2.scummvm` and an empty file `Monkey Island II.scummvm` then ES would show 2 entries: 
1. monkey2 -> this will work
2. Monkey Island 2 -> will not work

But this is not the fault of this PR it's the user fault ... the user should either rename the file to moneky2.scummvm or add a proper tag inside that is also available/setted into scummvm.ini